Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -2041,6 +2103,9 @@ def main():
         elif demisto.command() == 'cybereason-get-sensor-id':
             return_results(get_sensor_id_command(client, args))
 
+        elif demisto.command() == 'cybereason-get-machine-details':
+            return_results(fetch_machine_details_command(client, args))
```

Comment:
Please change the function name to align with the command name.

Reply:
Done